book.rb

```diff
@@ -0,0 +1,20 @@
+# book.rb
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] kindly  lets create  getter and setter methods for the rentals class by using `attr_accessor :title, :author, :rentals` instead of `  attr_reader :rentals` . This way, we'll be able to both read and write to the rentals attribute